driver/src/main/scala/com/stratio/sparkta/driver/factory/SparkContextFactory.scala

```diff
@@ -80,6 +84,7 @@ object SparkContextFactory extends SLF4JLogging {
 
     conf.setIfMissing("spark.streaming.concurrentJobs", "20")
     conf.setIfMissing("spark.sql.parquet.binaryAsString", "true")
+    conf.set("spark.streaming.receiver.writeAheadLog.enable", "true")
```

Comment:
Only when we provide the fault tolerance???

Read this:
However, these stronger semantics may come at the cost of the receiving throughput of individual receivers. This can be corrected by running more receivers in parallel to increase aggregate throughput. Additionally, it is recommended that the replication of the received data within Spark be disabled when the write ahead log is enabled as the log is already stored in a replicated storage system. This can be done by setting the storage level for the input stream to StorageLevel.MEMORY_AND_DISK_SER.